Update interfaces/web-animations.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/0ff6a6d/whatwg/idl/web-animations.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/536627667

foolip commented:
@lukebjerring I'm not able to inspect the results in detail, filed https://github.com/web-platform-tests/wpt.fyi/issues/1308. Can you take a look at that, and this PR when the issue is fixed?

autofoolip committed:
Update interfaces/web-animations.idl

Source: https://github.com/tidoust/reffy-reports/blob/0ff6a6d/whatwg/idl/web-animations.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/536627667
